salt/modules/reg.py

```diff
@@ -39,10 +39,10 @@
 
 # Import third party libs
 try:
-    from salt.ext.six.moves import winreg as _winreg  # pylint: disable=import-error,no-name-in-module
```

Comment:
I see changes removing winreg/_winreg are you moving away from winreg?

Reply:
Yes